pkg/registry/apps/daemonset/strategy.go

```diff
@@ -166,6 +166,7 @@ type daemonSetStatusStrategy struct {
 	daemonSetStrategy
 }
 
+// StatusStrategy is the default logic invoked when updating service status.
```

Comment:
copy/paste error - comments refer to "service status", can just change to "object status" (applies to all)

Reply:
Thank you for your review!